fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -464,6 +469,52 @@ public void execute() throws Exception {
 
                         handleQueryStmt(execPlan);
                         break;
+                    } catch (StarRocksConnectorException e) {
```

Comment:
Is it possible that some `StarRocksConnectorException` which isn't caused by expired meta cache arriving here? Which may cause the query is also retried when some non-recovable error occurs.

Reply:
it's impossible currently. But I have defined another type exception to catch it.